### ISSUE TITLE
[template] Add accessibility to statement about privacy and security

### ISSUE THREAD

**Issue by LJWatson** [w3c/charter-drafts#144]
We should add accessibility to the following statement:
<Each specification must contain a section detailing any known security or privacy implications for implementers, Web authors, and end users.<

Accessibility is a legal requirement in many regions of the world. The W3C should be ensuring that implementors and authors have the information they need to use our standards in ways that meet the accessibility needs of consumers.

<Each specification must contain a section detailing any known security, privacy, or accessibility  implications for implementers, Web authors, and end users.<

**Comment by svgeesus** [comment]
@LJWatson I notice this issue is still open. The [current template](https://w3c.github.io/charter-drafts/charter-template.html#coordination) says

> For all specifications, this (Working|Interest) Group will seek horizontal review for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the TAG. 

OK to close the issue?

It seems to have been resolved some time ago, but this issue was not updated to notify you of the fact.

**Comment by LJWatson** [comment]
That text is in the Co-ordination part of the template, but the text I was referring to was in the Success Criteria part.

The text in the Success criteria part of the template has been updated since I opened the issue, but it's a) quite lengthy, and b) puts privacy and security on a different footing to accessibility.

It'd be good to use the text we have in the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html) as the default: 
 
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

**Comment by svgeesus** [comment]
The part I was quoting is also in [success criteria](https://github.com/w3c/charter-drafts/blob/gh-pages/charter-template.html#L223) and there is an [open pull request](https://github.com/w3c/charter-drafts/pull/216#commitcomment-34236308) which would unify the two parts (privacy/security, and accessibility)

**Comment by LJWatson** [comment]
Perhaps I'm not reading the PR properly (always a possibility with a screen reader), but it still seems to have privacy and security in one paragraph (changed from SHOULD to MUST), and two paragraphs later an optional paragraph on accessibility (SHOULD still not MUST)?

Could we just make it simpler for all concerned, with the language from the WebApps charter?
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

**Comment by npdoty** [comment]
I appreciate the intent and the improved consistency of getting this into the default template.

We might want to say a little more than just that implications (for privacy, security, a11y, i18n) must be documented. In general in horizontal review, we try to address the issues that arise, not solely document them. I believe the template has language to that affect for a11y now, but privacy and security are currently in the "just document" state.

Anyway, this proposed resolution (didn't look to me like it was merged or that there was an open PR) would definitely be an improvement, and I'm happy to help keep wordsmithing if that would help. 
https://github.com/w3c/charter-drafts/commit/9e6e3e96b08194d34931ea40bfdc8d9fa269ff1c

**Comment by svgeesus** [comment]
The current template does still have two separate statements, firstly because the accessibility one starts with a conditional clause and secondly because it has more specific language:

```html
<p>Each specification should contain separate sections detailing 
  all known security and privacy implications for implementers, 
  Web authors, and end users.</p>

<p><i class="todo">For specifications of technologies that directly impact 
  user experience, such as content technologies, as well as protocols 
  and APIs which impact content: </i>
  Each specification should contain a section on accessibility 
  that describes the benefits and impacts, 
  including ways specification features can be used to address them,
  and recommendations for maximising accessibility in implementations.</p>
```

Starting from that text, @LJWatson and @npdoty what changes should be made?
